### DISCUSSION TITLE
Welcome to icd Discussions!

### DISCUSSION THREAD

**Discussion by jackwasey** [jackwasey/icd#209] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by twaters10** [comment]
Hey Jack, 

This ICD package is awesome and extremely useful for us! I'm currently just using the icd10cm2019 dataset. I was wondering if you plan on updating this to the 2022 codes, and if not could you direct me to how your creating/obtaining this dataset?

Thanks,
Taylor

**Comment by de-brown** [comment]
Hi Jack, 
I've noticed that this package is no longer available except through archives. Are there any plans to update it? It would be really helpful for my research. 

Thanks

**Reply by Lilyhappiness** [reply]
how can I even install it from Archives?

**Comment by simonbejjani** [comment]
hello dr, thank you for your input.
I am currently a master's student doing my thesis and i need to use the icd coding in french, in fact i have a database that contains a variable coded numerically by the icd10 but i can't retrieve the diseases' names and label them to their corresponding code , so my data is remaining like this 3659..... . i tried the explain table code , it didn't work . Can you please help me ?
